### PR TITLE
Feat/confirm sharing

### PR DESCRIPTION
**Pull request by Ldoppea**
When Alice share a password to Bob and this sharing has been accepted by Bob, then Alice should `confirm` the identity of Bob by checking his fingerprint phrase

This commit implements this feature and is based on mechanisms from cozy/cozy-libs#1340 (~~and so we have to wait this other PR to be merged before merging cozy-pass' one~~ this is now merged) 

~~Note: rejectUser is not implemented yet, this is my next task~~ : this is now implemented

### CODE REVIEW

**Comment by Ldoppea** [comment]
Rejection dialog has been implemented on cozy-pass-web but also on cozy-sharing : https://github.com/cozy/cozy-libs/pull/1340#issuecomment-903807260

File: src/cozy/wrappers/sharing/sharing.component.ts [hunk 1]
```diff
@@ -148,7 +148,18 @@ export class SharingComponent extends AngularWrapperComponent {
     }
 
     protected async rejectUser(user: User) {
-        // console.log('rejectUser', user)
+        try {
+            const client = this.clientService.GetClient();
+
+            await client.stackClient.fetchJSON(
```
Comment (Merkur39) [inline src/cozy/wrappers/sharing/sharing.component.ts:154]: Une raison particulière de ne pas utiliser `client.destroy` ?

Comment (Ldoppea) [reply src/cozy/wrappers/sharing/sharing.component.ts:154]: Oui, lorsque j'essaie d'utiliser destroy j'ai l'erreur que je t'ai montré l'autre jour : `ERROR Error: Uncaught (in promise): FetchError: {"error":"code=403, message=reserved doctype com.bitwarden.contacts unwritable"} FetchError: {"error":"code=403, message=reserved doctype com.bitwarden.contacts unwritable"}`

La route habituelle de gestion des doctypes ne semble pas branchée et il faut donc passer par `/bitwarden/contacts/:ID`.

Comment (Merkur39) [reply src/cozy/wrappers/sharing/sharing.component.ts:154]: Ok, je manque de connaissance sur `cozy-client`, mais c'est bon à savoir..

**Pull request merged**